app/views/provisioned.html

```diff
@@ -0,0 +1,156 @@
+<project-header class="top-header"></project-header>
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header header-toolbar">
+        <div class="container-fluid">
+          <div class="page-header page-header-bleed-right page-header-bleed-left">
+            <h1>
+              Provisioned Services
+              <span class="page-header-link">
+                <a ng-href="{{'services' | helpLink}}" target="_blank">
```

Comment:
We need to link to the real service catalog topic, not to the services topic, which is something different. You might just remove this for now.